integration/dashboards_test.go

```diff
@@ -334,6 +334,68 @@ func createGraphWithLogQuery() []datadog.Graph {
 	return graphs
 }
 
+func createGraphWithRumQuery() []datadog.Graph {
```

Comment:
Thanks for these. These should be called inside a method `Test...` similar to https://github.com/zorkian/go-datadog-api/blob/f06edcd25563ca9a485d70a035c372640815a5c7/integration/dashboards_test.go#L134